logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/backup/RestoreWebBackupUseCaseTest.kt

```diff
@@ -128,7 +128,7 @@ class RestoreWebBackupUseCaseTest {
         verify(arrangement.migrationDAO)
             .suspendFunction(arrangement.migrationDAO::insertConversation)
             .with(any())
-            .wasInvoked()
+            .wasNotInvoked()
```

Comment:
Is that correct you changed it from `wasInvoked` to `wasNotInvoked`

Reply:
the test is for unsupported Conversations, so the insert should not be inserted, and the `wasInvoked()` before the update did literally nothing.